### PR TITLE
Offline bundle

### PR DESCRIPTION
**Pull request by pjanik**
@scytacki, it's based on your idea. I didn't modify JSmol.full.js but monkey patched `Jmol._getFileData` instead. Hopefully readme explains other bits.

I also deploy this package to S3, so it can be used by http://lab.concord.org/standalone/create.html page.

### CODE REVIEW

File: embeddable.html [hunk 1]
```diff
@@ -38,6 +38,12 @@
 <script type="text/javascript" src="jsmol/JSmol.min.js"></script>
 <script type="text/javascript" src="iframe-phone/dist/iframe-phone.js"></script>
 <script type="text/javascript" src="shutterbug.js/dist/shutterbug.js"></script>
+<!--
+  Do not remove comment before. It's used by offline-bundle app.
```
Comment (scytacki) [inline embeddable.html]: Should 'before' be changed to 'below'?

**Review by scytacki — APPROVED** [review]
Looks great to me.

**Comment by scytacki** [comment]
One thing I didn't fully understand was the binary file handling. They are just copied, right?  
So how are images handled? Does JSMol create `img` tags or use canvas elements, and those things work with file URLs?  
I thought non image binary files were loaded by JsMol using its proxy service, and that would fail when offline. Or do I have this all wrong?

**Comment by pjanik** [comment]
> One thing I didn't fully understand was the binary file handling. They are just copied, right?
So how are images handled? Does JSMol create img tags or use canvas elements, and those things work with file URLs?

So, binary, non-image files are not handled. I just skip them for now. We don't use any at all, so I don't have anything to play with. Hopefully we could handle them if we convert them to base64 (JSmol logic is more complex, but that's what I'd hope for if we ever needed that).

Images are handled separately. My guess is that JSmol loads them using Image() class, not using _getFileData - `file://` protocol is supported anyway. So, I just copy all the images from `/models` dir and keep the original paths. It should work, I've tested that.

Actually, I might change approach later. Instead of bundling everything into HMTL, I might try to generate JS files with something like `Jmol._fileData['models/dna/dna.pdb'] = '<file_content>'`. It might make it easier to add new, custom model files. We might need it for PASCO JSmol template interactive. Currently, they'd have to edit 20MB HTML file - it take a while to open it in editor and it's not very readable. The only problem is that it'd be nice to use new js strings (``) to handle multiline strings easily, but we'd support less browsers (probably IE can't handle it).

**Comment by pjanik** [comment]
I've fixed some small bugs, so I'm going to merge it now.

**Pull request merged**